### ISSUE TITLE
In the toolkit: Signup asks only for a name, while Login asks for an email or username

### ISSUE THREAD

**Issue by kerryboyko** [userfront/userfront-react#28]
I'd love to use the default signup forms in the toolkit, they have everything I need, but there is one large problem:

The signup form has the following fields:
* Name -> user.name  (Example: "kerry")
* Email -> user.email  (Example: "kerry@domain.com"
* Password -> user.password (hashed and salted, of course)

While the login form has the following fields:
* Email or Username -> user.email or user.username
* Password => user.password

Because user.name !== user.username, if I were to try to login with "kerry" - it would fail, as there would be no username by that name.  If I were to login with 'kerry@domain.com" it would succeed.  

Basically, is there any way to: 

A) Alter the form so that the name given is also the username
B) Alter the form so that it *only* asks for the username
C) Make the name equal to the username via a webhook.

**Comment by RJFelix** [comment]
Hi @kerryboyko, the forms currently can't be modified in this way, but it's an excellent point and I agree - thank you for bringing it to our attention! I'll add this to our roadmap to be addressed in future updates to our React Toolkit.

Option C is possible as a workaround in the meantime, with a little bit of glue code:

1. Add a [User Created webhook](https://userfront.com/dashboard/webhooks) for your account, to call an endpoint on your server or serverless infrastructure
2. Read [the incoming user record sent with the webhook](https://userfront.com/docs/webhooks#user-created) to get `user.name` (`record.name` in the webhook body)
3. Call the [Update User endpoint](https://userfront.com/docs/api#update-user) from the Server-to-Server API to set the `user.username` field to that value, using an [admin API key](https://userfront.com/dashboard/api-keys) to authenticate

I hope this helps. Feel free to ping me with any other questions or concerns!

**Comment by kerryboyko** [comment]
Thanks, and yes, it would be good, but I'm already halfway to converting to FusionAuth, which I think is a slightly better fit for this project (mostly because this is a "learning project" greenfield and FusionAuth allows you to host your own DB on docker.  

If this ever becomes a real product, (which it could, never know), I'll be evaluating both solutions.

**Comment by RJFelix** [comment]
We're tracking this in our Linear, DEV-389. I'm closing this issue because we're deprecating this repo and moving to https://github.com/userfront/toolkit
